htdocs/compta/prelevement/class/bonprelevement.class.php

```diff
@@ -1790,7 +1790,9 @@ public function generate($format = 'ALL', $executiondate = 0, $type = 'direct-de
 				if ($result != -2) {
 					$fileEmetteurSection .= $this->EnregEmetteurSEPA($conf, $date_actu, $nbtotalDrctDbtTxInf, $this->total, $CrLf, $format, $type, $fk_bank_account);
 				}
-
+				if($conf->global->SEPA_ROUND_TWO_ZERO){
+					$this->total=number_format($this->total,2,".","");
```

Comment:
We already have a functions to format price using the number if decimal if the country.
Can you try to use price2num with param 'MT' that can be used to force the rounding .

Reply:
As you have told me, I have used the price2num function with the 'MT' parameter but I have also had to use the number_format function since this function, when passing a value like 6.50, returns 6.5 and I need it to return 6.50 as i explained to you at the beginning of the request